beers/models.py

```diff
@@ -80,3 +80,99 @@ class Manufacturer(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Beer(models.Model):
+    name = models.CharField(max_length=25, db_index=True)
+    style = models.ForeignKey(
+        BeerStyle, models.DO_NOTHING, related_name='beers',
+        # TODO: prevent this being null?
+        blank=True, null=True,
+    )
+    manufacturer = models.ForeignKey(
+        Manufacturer, models.CASCADE, related_name='beers',
+    )
+    in_production = models.BooleanField(default=True)
+    abv = models.DecimalField(
+        'Alcohol content (% by volume)',
```

Comment:
Can we add these descriptions to the `BeerStyle` model? (Either here or in followup PR, up to you)

Reply:
Follow-up PR works for me.